lottery_categories/db.py

```
@@ -0,0 +1,115 @@
+from pymongo.collection import Collection
+from pymongo.results import InsertOneResult, UpdateResult, DeleteResult
+from pymongo.errors import PyMongoError
+from bson import ObjectId
+from typing import List, Dict, Any, Optional
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import

(F401)